packages/mdc-toolbar/README.md

```diff
@@ -1,8 +1,7 @@
 # MDC Toolbar
 
-MDC Toolbar provides a RTL-aware Material Design toolbar component adhering to the
-[Material Design toolbar spec](https://www.google.com/design/spec/components/toolbars.html)
-Toolbars scroll with content by default, but supports fixed on
+MDC Toolbar provides a multiple row container for application navigation, title
```

Comment:
Or how about 

"MDC Toolbar is a multiple row containers for items such as application title, navigation menu, and tabs, among other things."

Since `mdc-toolbar` itself is a container which contains `mdc-toolbar__row`.